about.html

```diff
@@ -0,0 +1,113 @@
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </div>
+        <div class="main-nav">
+          <div class="logo"><a href="index.html"><img src="images/logo.png" alt="logo"></a></div>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#sponsor">Sponsor</a></li>
+          </ul>
+        </div>
+        <div class="nav-bar">
+          <div class="hamburger">
+            <img src="icons/mobile_menu.png" alt="hamburger">
+          </div>
+          <div class="mobile-nav hide">
+            <ul>
+              <li class="cross-icon"><img src="icons/cross_icon.svg" alt="cross icon"></li>
+              <li class="menu-item"><a href="index.html">Home</a></li>
+              <li class="menu-item"><a href="#">Program</a></li>
+              <li class="menu-item"><a href="#sponsor">Sponsor</a></li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section class="headline-section">
+      <div class="headline-content">
+        <p class="main-color">"Hello! Sharing World"</p>
+        <h1 class="text-center main-color"> JSWORLD CONFERENCE 2023 </h1>
+        <div class="description-container">
+          <p class="text-center">A 3-day festival where 1000+ JavaScript developers from different countries are joining to share the latest technological updates, ideas and creativity in February 2023 in Islamabad, the capital city of Pakistan.</p>
+        </div>
+        <div class="contact text-center">
+          <p>Please contact us for further questions about
+            JSWORLD Conference 2023!</p>
+          <a href=""><i class="fa-brands fa-whatsapp"></i><p><u>+31622227889</u></p></a>
+        </div>
+      </div>
+    </section>
+  </div>
+  <div class="half-02">
+    <section class="logo-competition">
+      <h2 class="text-center">JSWORLD CONFERENCE 2023 logo</h2>
+      <div class="underline"></div>
+      <p class="text-center">The logo of JSWORLD Conference 2023, was decided through competition held from 7th to 9th july.</p>
+      <div class="conference-logo">
+        <img src="images/logo.png" alt="">
+      </div>
+    </section>
+    <section class="past-events">
+      <h2 class="text-center">See the past JSWORLD Conferences</h2>
+      <div class="underline"></div>
+      <p class="text-center">Take a look at the recent JSWORLD Conferences.</p>
+      <div class="past-events-img">
+        <div>
+          <img src="images/past-event-images/Js-1.jpg" alt="past events">
+          <h3 class="text-center">2019</h3>
+          <p class="text-center">JSWORLD Conference 2019</p>
+        </div>
+        <div>
+          <img src="images/past-event-images/Js-2.jpg" alt="past events">
+          <h3 class="text-center">2020</h3>
+          <p class="text-center">JSWORLD Conference 2020</p>
+        </div>
```

Comment:
I think you did a great job with these images. I watched your presentation video and I know you tried very hard to make this section. However, as code reviewers, we have to check a list of requirements and this is one of them. Kindly make this section stick as much as possible to the template provided. :pray:  :pray: . I think you could try adding the image as a background using CSS and changing the opácity property. :+1: :smiley: :smiley: 
![image](https://user-images.githubusercontent.com/98363075/196827743-edc39a2d-dd40-4f1c-8af6-c8d34243cbfa.png)